others/py3compat.py

```diff
@@ -0,0 +1,7 @@
+# Workaround for Python3 compat
+import sys
+if sys.version_info > (3,):
```

Comment:
https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection
```python
try:
    long        # Python 2
except NameError:
    long = int  # Python 3
```

Reply:
good one, i'll chage that